app/models/flowmanagement/RemoveServiceFlowModel.scala

```diff
@@ -16,6 +16,15 @@
 
 package models.flowmanagement
 
-case class RemoveServiceFlowModel(dummy: Int)
+import models.businessmatching.BusinessActivity
+import play.api.libs.json.Json
 
-object RemoveServiceFlowModel
+case class RemoveServiceFlowModel(existingActivities: Option[Set[BusinessActivity]] = None,
+                                  activitiesToRemove: Option[Set[BusinessActivity]] = None)
```

Comment:
Is `existingActivities` being used, and could we not simply retrieve this information from the business matching data?